pkg/shim/scheduler.go

```diff
@@ -241,16 +242,23 @@ func (ss *KubernetesShim) doScheduling(e *fsm.Event) {
 
 func (ss *KubernetesShim) registerShimLayer() error {
 	configuration := conf.GetSchedulerConf()
+
+	var buildInfoMap map[string]string
+	buildInfoMap["version"] = conf.Version
+	buildInfoMap["date"] = conf.Date
+
 	registerMessage := si.RegisterResourceManagerRequest{
 		RmID:        configuration.ClusterID,
 		Version:     configuration.ClusterVersion,
 		PolicyGroup: configuration.PolicyGroup,
+		BuildInfo:   buildInfoMap,
 	}
 
 	log.Logger().Info("register RM to the scheduler",
 		zap.String("clusterID", configuration.ClusterID),
 		zap.String("clusterVersion", configuration.ClusterVersion),
-		zap.String("policyGroup", configuration.PolicyGroup))
+		zap.String("policyGroup", configuration.PolicyGroup),
+		zap.String("buildInfo", fmt.Sprintf("%v", buildInfoMap)))
```

Comment:
use the `zap.Any("buildInfo", buildInfoMap)` construct and let zap figure out how to log this.